### DISCUSSION TITLE
Error during evaluation

### DISCUSSION THREAD

**Discussion by ibrahim014** [open-mmlab/mmyolo#561] [General]
Hi,

I am trying to train the Yolov5 model using a custom dataset (containing 1 class). The model training went successfully. However, when it comes to evaluating the model I am getting the following error:

 data['category_id'] = self.cat_ids[label]
IndexError: list index out of range

I tried to change the # of classes in cooc.py file but got the same error. Can anyone know where I need to make changes?

**Comment by hhaAndroid** [comment]
@ibrahim014 Your metainfo field is wrong, please refer to https://github.com/open-mmlab/mmyolo/blob/main/projects/misc/custom_dataset/yolov5_s-v61_syncbn_fast_1xb32-100e_cat.py#L27 or https://github.com/open-mmlab/mmyolo/blob/main/docs/zh_cn/user_guides/custom_dataset.md
